clang/include/clang/Analysis/FlowSensitive/ASTOps.h

```diff
@@ -146,6 +146,10 @@ struct ReferencedDecls {
   /// Free functions and member functions which are referenced (but not
   /// necessarily called).
   llvm::DenseSet<const FunctionDecl *> Functions;
+  /// Parameters of other functions, captured by reference by a lambda. This is
```

Comment:
I was confused before reading the code. It might be clearer to mention a "surrounding function" rather than "other functions". Maybe something like:
```
When analyzing a lambda's call operator, the set of all parameters (from the surrounding function) that the lambda captures by reference.
```

Reply:
I apparently forgot to update this after including captures by value, too. Updated.